frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.module.css

```diff
@@ -0,0 +1,12 @@
+.triggerButton {
```

Comment:
This is the exact same as @heypoom 's version, but in CSS module

https://github.com/metabase/metabase/blob/9a05331b469d5324ee5614eda616478ab551d2b0/frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.styled.tsx#L21-L29